package.json

```diff
@@ -132,6 +132,7 @@
   },
   "devDependencies": {
     "@types/mocha": "^5.2.7",
+    "@types/proper-lockfile": "^4.1.1",
```

Comment:
This should be a regular dependency since automation-client is a dependency to other packages.

Reply:
Updated.